All build tests failing

Currently all build tests from Travis CI are failing on the planck circuit layout. [Example here](https://travis-ci.org/jackhumbert/qmk_firmware/builds/153217928).

```
Compiling planck/planck with circuit                                                                   [ERRORS]
../../keyboards/planck/keymaps/circuit/keymap.c:46:29: error: conflicting types for ‘tap_dance_actions’
 const qk_tap_dance_action_t tap_dance_actions[] = {
                             ^
In file included from ../../quantum/quantum.h:60:0,
                 from ../../keyboards/planck/planck.h:11,
                 from ../../keyboards/planck/keymaps/circuit/keymap.c:3:
../../quantum/process_keycode/process_tap_dance.h:53:30: note: previous declaration of ‘tap_dance_actions’ was here
 extern qk_tap_dance_action_t tap_dance_actions[];
                              ^
 | 
 | 
 | 
make[1]: *** [../../.build/obj_planck_rev4_circuit/keyboards/planck/keymaps/circuit/keymap.o] Error 1
make: *** [/keyboards/planck-all] Error 1
```

This should be fixed asap.

--- a/keyboards/planck/keymaps/circuit/keymap.c
+++ b/keyboards/planck/keymaps/circuit/keymap.c
@@ -43,7 +43,7 @@ enum planck_keycodes {
 
 // Tap Dance Definitions
 #ifdef TAP_DANCE_ENABLE
-const qk_tap_dance_action_t tap_dance_actions[] = {
+qk_tap_dance_action_t tap_dance_actions[] = {
 	[0]  = ACTION_TAP_DANCE_DOUBLE(KC_LSFT, KC_CAPS)
 };
 #endif

